config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-postsubmits.yaml

```diff
@@ -1,6 +1,7 @@
 postsubmits:
   kubernetes-sigs/node-feature-discovery:
   - name: postsubmit-node-feature-discovery-verify-master
+    cluster: k8s-infra-prow-build
```

Comment:
this one should probably be on the EKS prow, and it must have been moved there previously as it's using `public.ecr.aws/docker/library/golang:1.22` 

it's not doing e2e is it?

Reply:
https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/scripts/test-infra/verify.sh 
so yeah, not e2e

Reply:
should I move it to EKS?

Reply:
yes

Reply:
done